Gameplay/fr.md

```diff
@@ -0,0 +1,101 @@
+| C    | 70-79% de précision                                                           |
+| D    | 60-69% de précision                                                           |
+| F    | Echec                                                                         |
+
+#### Précision
+
+La précision est une moyenne pondérée des appréciations reçues par rapport à la somme des appréciations tout au long d'une partie.
+
+Vous recevez une appréciation par note "normale", et deux par note "longue" (appuyer puis relâcher) ("Slider").
+
+| Appréciation | Poids  |
+|--------------|--------|
+| Marvelous    | 100%   |
+| Perfect      | 98.25% |
+| Great        | 65%    |
+| Good         | 25%    |
+| Okay         | -100%  |
+| Miss         | -50%   |
+
+#### Calcul de score
+
+Votre score est une évaluation de votre performance tout au long d'une partie en fonction de la difficulté de la map. C'est le même calcul qui est utilisé pour calculer les scores dans les classements mondiaux.
+
+La formule est relativement simple, c'est la difficulté d'une map multiplié par votre précision sur 98, puissance 6; ou mathématiquement :
+
+`DifficultyRating * Math.Pow(Accuracy / 98, 6)`
+
+#### Classement Global
+
+Votre classement global est un total pondéré de toutes vos parties joué dans le mode de jeu "de touches" (4K ou 7K). Le 4K et 7K sont séparés et ont chacun un classement différent.
+
+Seulement votre meilleur score sur une map compte, et surpasse toutes les autres parties sur cette même map.
+
+Le calcul du total pondéré est similaire aux Points de Performances (PP), utilisés dans le jeu de rythme osu! :
+
+`overall = scoreRating[1] * 95^0 + scoreRating[2] * 95^1 + scoreRating[3] * 95^2 + ...`
+
+#### Score
+
+Le score est une autre mesure de vos performances en jeu dans Quaver. C'est principalement une manière de mesurer à quelle fréquence vous êtes capable de tenir un combo en corrélation avec les appréciations obtenues. Ce système n'est utilisé dans aucun classement global, et est dans le jeu purement sous une forme d'"xp". Chaque map à un score maximum de 1 million de points.
+
+#### Fenêtre de jugement d'erreurs relatives au temps (Judgement Timing)
+
+Cette section explique la manière dont est utilisée la fenêtre de jugement qu'utilise Quaver. Elle détaille le temps en millisecondes nécessaire pour appuyer sur une touche pour une note dans le but de la taper au bon moment. Actuellement, il y a une fenêtre de jugement d'erreurs relatives au temps dans Quaver. Cependant, cela pourrais changer dans le future.
+
+| Appréciation | Délai  |
+|--------------|--------|
+| Marvelous    | ±18ms  |
+| Perfect      | ±43ms  |
+| Great        | ±76ms  |
+| Good         | ±106ms |
+| Okay         | ±127ms |
+| Miss         | ±164ms |
+
+Si lors du relâchement d'une longue note (Slider), vous obtenez l'appréciation `Okay`, cela est bien pire qu'un raté (miss) pour le calcul de la précision.
```

Comment:
remove Slider
we don't need to reference osu!. If you want to be more precise, reference it as a **hold note** (note maintenue)